### DISCUSSION TITLE
new menue settings getting lost after programming channels

### DISCUSSION THREAD

**Discussion by mentanah** [egzumer/uv-k5-firmware-custom#243] [Q&A]
first of all, dear egzumer and all dedicated, thanks a lot for this brilliant repository and the excellent development of the custom firmware. I cant wait for release 0.2 since there are more than 50 new commits.

By programming the channels and change the settings with original pc software, some new settings (battery type, backlight min max settings) are getting lost. Is it possible to preserve them?

**Comment by kittenarmy** [comment]
Which PC software are you using? I suggest using [CHIRP](https://chirp.danplanet.com/projects/chirp/wiki/ChirpNextBuild) if you are not already. You can update channels and settings independently.

**Comment by mentanah** [comment]
Thank you for this hint. The Radio ist already fully implemented in chirp?
I was using the Portable Radio CPS from Quansheng or Radtel.

**Comment by egzumer** [comment]
Use CHIRP, don't open settings tab!!!
The driver specifically for this firmware is being developed.

**Comment by ErikS-web** [comment]
quite simple... just do...

Radio - Download from Radio.
DO_NOT enter Tab Settings !
Modify Channels 
Radio - Upload to Radio

Enjoy.
